ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -641,6 +641,21 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-nested-virtualization-osdisk-type-size
+- as: gcp-ipi-sno-tp-arm-f28-ota
+  cron: 35 22 4 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      ENABLE_OTA_TEST: OCP-32747
+      FEATURE_SET: TechPreviewNoUpgrade
+      OCP_ARCH: arm64
+    test:
+    - ref: cucushift-ota-preupgrade
+    - ref: cucushift-ota-postupgrade
```

Comment:
There is not upgrade happen, so postupgrade sounds not necessary now, the checkpoints could be moved into one step.

Reply:
Sounds reasonable, I will update it